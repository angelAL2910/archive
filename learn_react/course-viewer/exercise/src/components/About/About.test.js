import React from "react";
import renderer from "react-test-renderer";
import About from "./About";

test("Tests for the About component", () => {
        let tree = renderer.create(
            <About />
        ).toJSON()

        expect (tree).toMatchSnapshot();
})