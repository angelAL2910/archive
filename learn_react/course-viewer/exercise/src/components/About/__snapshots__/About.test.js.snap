// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the About component 1`] = `
<div
  className="element"
>
  About
</div>
`;
